.gitignore

```
@@ -8,3 +8,8 @@ wwwroot/
 node_modules/
 *.user
 .pnpm-debug.log
+/.editorconfig
+/TestSolutions/Lombiq.Analyzers.*/.editorconfig
```

You can generalize this to:
```suggestion
/TestSolutions/**/.editorconfig
```

---

I used this to be similar as the line below, where it actually matters. (because `/**/` applies to `.` as well)